src/implementations/vanilla/components/select/select.js

```diff
@@ -923,8 +943,11 @@ export class Select {
         break;
 
       case 'Tab':
-        e.preventDefault();
-        this.input.focus();
+        if (e.shiftKey) {
+          this.closeButton.previousSibling.focus();
+        } else {
+          this.input.focus();
```

Comment:
a normal select would also collapse the dropdpwn when tabbing on the last element (not the same, in that case is an option), i wonder if we shouldn't do the same.

Reply:
we may keep the default behaviour for the tab, as you suggested, and keep the custom one for the arrows... I'll do a test

Reply:
There are already some differences between Chrome and Firefox here; Firefox adds some uneeded focus elements when using tab.
I didn't check with other browsers, but as we already have the logic in place to handle the focus with arrow, it would be safer to use it also for tab